util/java/libtiled-java/src/main/java/org/mapeditor/io/TMXMapReader.java

```diff
@@ -108,7 +110,7 @@ String getError() {
         return error;
     }
 
-    private static String makeUrl(String filename) throws MalformedURLException {
+    private static String makeUrl(String filename) {
```

Comment:
Since every invocation of this `makeUrl` member now goes like `new URL(makeUrl(...))`, what about making this function return a `URL` instance?

Reply:
Nice catch!